python/mxnet/foo/nn/layer.py

```diff
@@ -207,6 +222,80 @@ def generic_forward(self, F, x, *args, **kwargs):
         raise NotImplementedError
 
 
+class _Binary(Layer):
```

Comment:
I have removed these layers completely. Although they are useful, they might not be very intuitive for users. I will write a more detailed guide for legacy layers later.